File: lua/refactoring/tests/refactor/115/c/example/inline_func.start.c
#include <stdio.h>

int foo(int a) {
  int i = 3;
  printf("%d %d", i, a);
  return i;
}


int simple_function(int a) {
  printf("this is a test\n");
  int i = foo(a);


  return i;
}

File: lua/refactoring/tests/refactor/115/python/example/inline_func.start.py
def foo_bar(a, test):
    test_other = 11
    for idx in range(test - 1, test_other):
        print(idx, a)
    return test_other


def simple_function(a):
    test = 1
    test_other = foo_bar(a, test)

    return test, test_other
